scripts/devaccess/test_devget.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python
+
+''' Tests for devaccess_wrap.py tool '''
+
+# Removing invalid variable names for tests so that I can
+# keep them brief
+# pylint: disable=invalid-name
+
+# Allow modifying internal structures for mocking
+# pylint: disable=protected-access
+
+# 'mock' not available in jenkins environment
+# pylint: disable=import-error
+
+import os
+import sys
+import unittest
+from mock import MagicMock, mock, patch
+
+module_path = os.path.join('/'.join(os.path.realpath(__file__).split('/')[:-1]))
+sys.path.insert(0, module_path)
+
+# Need to set module path (using oc.*) before trying to
+# import devaccess_wrap
+# pylint: disable=wrong-import-position
+from devaccess_wrap import DevGet
+
+class DevGetTest(unittest.TestCase):
+    ''' unittest class for testing devaccess_wrap.py '''
+
+    # No 'self' use is intended here
+    # pylint: disable=no-self-use
+    def setUp(self):
+        ''' steps to run before every test run '''
+        DevGet.CONFIG_FILE = './test_devaccess_config.yaml'
+        sys.argv = ['devaccess_wrap', 'READ_SSH', 'nobody']
+
+    def test_unsupported_command(self):
+        ''' Test trying to run a command that isn't supported '''
+        os.environ['SSH_ORIGINAL_COMMAND'] = '/usr/bin/blah'
```

Comment:
more test cases:

same token in two arguments:  -n json -o json
no namespace given:  oc get foo -n
invalid namespace given:  oc get foo -n (long string, string with % chars, string with path traversal, etc)